torch/csrc/distributed/c10d/init.cpp

```diff
@@ -291,6 +293,7 @@ They are used in specifying strategies for reduction collectives, e.g.,
   shared_ptr_class_<::c10d::FileStore>(module, "FileStore", store)
       .def(py::init<const std::string&, int>());
 
+#ifndef _WIN32
```

Comment:
we will also need this for TCP store?

Reply:
TCP store does not be supported for first step, so I disable TCP store from python script also.